library/src/index.js

```diff
@@ -47,6 +49,38 @@ window.CoNDeT = function (configs) {
 
     return new window.CoNDeT.ui.Mode({});
   }
+
+  function addTable() {
+    if (common.mode === window.CoNDeT.ui.DisplayMode) return;
+
+    var tableObj = {};
+    var rowTab = [];
+    var id = prompt("Enter Table ID:");
+    var name = prompt("Enter Table Name:");
+    var classType = prompt("Enter Table Class:");
+
+    for (var i = 0; i < 3; i++) {
+      rowTab.push({
+        conditions: [],
+        decisions: [],
+        connections: [],
+        row_id: id + "_row_" + i,
+      });
+    }
+
+    tableObj.id = id;
+    tableObj.name = name;
+    tableObj.class = classType;
+    tableObj.coordinates = {};
+    tableObj.coordinates.x = 0;
+    tableObj.coordinates.y = 0;
+    tableObj.columns = {};
+    tableObj.columns.conditions = ["defaultCon", ""];
+    tableObj.columns.decisions = ["defaultDet", ""];
+    tableObj.rows = rowTab;
+
+    common.stateModifier.createTable(tableObj);
+  }
```

Comment:
Można przerobić, aby funkcja przyjmowała jeden parametr (common) i przenieść do core. W tym pliku powinno być jak najmniej, bo jest on na wysokim poziomie abstrakcji, a ta funkcja jest niskopoziomowa

EDIT:  
Ponieważ podajesz ją jako `addTable: addTable,` to trzeba zrobić function factory która jest funkcją i przyjmuje `common` i zwraca tą funkcję